framework/python/src/common/session.py

```diff
@@ -501,29 +501,15 @@ def update_profile(self, profile_json):
 
     else:
 
+      # Update the profile
+      risk_profile.update(profile_json)

```

Comment:
I get this error: TypeError: RiskProfile.update() missing 1 required positional argument: 'profile_format'

Reply:
Resolved.